app/models/katello/upstream_consumer.rb

```diff
@@ -0,0 +1,21 @@
+require 'katello/resources/candlepin'
+
+module Katello
```

Comment:
Would be curious what you and others think about this - in PR7241 I opted to not use this class and called the resource directly from my remove action

Reply:
Right now, it just handles the translation between the controller params (what the user interacts with) and the resource. It's also where the loop is for each pool, and that's going to be removed once the Candlepin API is fixed upstream, so it'll really just be doing the param translation after that. I don't see a reason to keep it; we can move map_pool_attributes to the controller.